google/resource_compute_instance_group_manager.go

```diff
@@ -419,7 +419,7 @@ func resourceComputeInstanceGroupManagerRead(d *schema.ResourceData, meta interf
 	}
 
 	manager, err := getManager(d, meta)
-	if err != nil {
+	if err != nil || manager == nil {
```

Comment:
This additional condition will cause some unexpected results, I think - you'll return `nil` as the error, where I think you probably want something like `fmt.Errorf("Instance Group Manager %s does not exist", d.Id())`.  What do you think?

Reply:
If we return an error from the `getManager` function, this will be returned here in `resourceComputeInstanceGroupManagerRead` and terraform will display the error to the user. 
 In order for `terraform plan` to work `resourceComputeInstanceGroupManagerRead` needs to return `nil`.

Reply:
Aha, you do the `d.SetId("")` in the `getManager` function, I see it now.  Yes, you are right.